### ISSUE TITLE
It doesn't work

### ISSUE THREAD

**Issue by mig4ng** [ahung89/bomb-arena#3]
Hey!
I tried to clone your game and run it locally, but as soon as I change the client socket (in Main.js) to localhost:8000, I can create games but inside the game I can't move or place bombs, but when I try to use the socket of you current application with a local client it works, so I assume the current server you have on github is not updated with last changes you made on it.
Can you try to help me solve this?
Thanks!

**Comment by ahung89** [comment]
Hey, sorry about the delayed response. Github doesn't notify me when people open issues on my repos, so I only saw this because I was randomly poking through my old code.

The code in the repo is up to date and is the same code that's running on heroku. Could you describe exactly how you are trying to run the game? It sounds to me like you might not actually have started running the server. I might need to add some instructions to the README be more clear, but you have to actually run the node server (run the file server.js). If the node server isn't running then none of the serverside logic (bombs, etc.) will work.

**Comment by mig4ng** [comment]
I cloned it with git, installed dependencies and ran grunt, everything is working except the movement and bomb placement, but I can create and join sessions

**Comment by ahung89** [comment]
I just re-cloned on my new PC to make sure things are working and I think I know what's going on with your machine. These are the steps to run the game locally:
1) Make sure you have nodeJS installed (https://nodejs.org/en/download/)
2) Go to main.js and make sure the line socket = io("http://localhost:8000") is uncommented (sounds like you've already done this). Comment out the line above it (line 10) that is pointing to Heroku.
3) From the command line, navigate to the root directory and execute this command:
node server/server.js
4) Once the command is running, you shouldn't see any output in your terminal (it should just look like its hanging - this means the server has started). Now, open up your browser and navigate to localhost:8000/index.html. You should be able to host and join games from your machine.

Let me know if you still have trouble.

**Comment by ahung89** [comment]
Thanks for bringing this to my attention. I figured out what the problem is and I pushed a fix. Try pulling the latest commit, reinstall dependencies, and see if you still have problems.

Basically, the package.json had a ^ in front of the socket.io dependency, causing it to pull in the latest version of the package. I think one of the recent releases of socket.io changed the way socket IDs are assigned (started appending a /# to the front). But my clientside code was pulling in an earlier package that didn't add this prefix. So basically the player IDs were inconsistent between the client and server and the player object never got instantiated. That's why none of the actions were working. I tested the fix and the packages are in sync now.

Anyway, let me know if you have more problems.
